src/core/memory.cc

```diff
@@ -1548,25 +1548,25 @@ void configure(std::vector<resource::memory> m, bool mbind,
     }
 }
 
-statistics stats() {
+statistics stats() noexcept {
     return statistics{alloc_stats::get(alloc_stats::types::allocs), alloc_stats::get(alloc_stats::types::frees), alloc_stats::get(alloc_stats::types::cross_cpu_frees),
         cpu_mem.nr_pages * page_size, cpu_mem.nr_free_pages * page_size, alloc_stats::get(alloc_stats::types::reclaims), alloc_stats::get(alloc_stats::types::large_allocs),
         alloc_stats::get(alloc_stats::types::foreign_mallocs), alloc_stats::get(alloc_stats::types::foreign_frees), alloc_stats::get(alloc_stats::types::foreign_cross_frees)};
 }
 
-size_t free_memory() {
+size_t free_memory() noexcept {
```

Comment:
This throws when seastar is compiled wit default allocator.

Reply:
where?

Reply:
This does not have a non-seastar alloc definition it appears.